Specs/DateTime/English/DateTimePeriodExtractor.json

```diff
@@ -122,6 +122,8 @@
   },
   {
     "Input": "I'll be out from 2:00pm, 2016-2-21 to 3:32, 04/23/2016",
+    "Comment": "Java does not correctly handle lookbehinds.",
```

Comment:
It depends on the Java version, no?
Which SDK and version did you use?

Reply:
I am using openjdk version 15. 
But the problem is that the java code use the regex matcher defined in RegExpUtility (e.g. in BaseDateTimePeriodExtractor.java at line 71: Match[] matches = RegExpUtility.getMatches(regex, input);)
and this matcher wrongly handles negative lookbehinds. 
(It first looks for matches of the negative lookbehind group and if found, it searches for matches of the following regex group without taking into account its position in the full regex. 
So, for example, the regex (?<month>\d{2})/(?<!,\s)(?<year>\d{2}) fails to match the input "So, 12/20" because "12" matches the "year" group regex and it follows a comma)

Reply:
Can you open a Java-specific issue for this and describe the problem? As the Java packages are not officially out, we can look into it before releasing them.

Reply:
ok, will do.